dependencies.props

```diff
@@ -91,16 +92,17 @@
       <ElementName>CoreClrExpectedPrerelease</ElementName>
       <BuildInfoName>CoreClr</BuildInfoName>
     </XmlUpdateStep>
-    <XmlUpdateStep Include="External">
-      <Path>$(MSBuildThisFileFullPath)</Path>
-      <ElementName>ExternalExpectedPrerelease</ElementName>
-      <BuildInfoName>External</BuildInfoName>
-    </XmlUpdateStep>
     <XmlUpdateStep Include="Standard">
       <Path>$(MSBuildThisFileFullPath)</Path>
       <ElementName>NETStandardPackageVersion</ElementName>
       <PackageId>$(NETStandardPackageId)</PackageId>
     </XmlUpdateStep>
+<!--
+    <XmlUpdateStep Include="External">
+      <Path>$(MSBuildThisFileFullPath)</Path>
+      <ElementName>ExternalExpectedPrerelease</ElementName>
```

Comment:
Good catch that these need to be commented out. I think they could be conditioned on whether the `RemoteDependencyBuildInfo` exists, but that seems more complicated than it's worth.